db/migrate/20190129210934_update_countries.rb

```diff
@@ -34,11 +57,18 @@ def self.field_name_map
     end
 
     def self.from_hash(in_hash)
-       if in_hash[:name]
-         in_hash[:name] = adjust_country_name(in_hash[:name])
-         existing = self.where("lower(name) like ?", in_hash[:name].downcase).first || self.new()
-         existing.update_attributes(in_hash)
-       end
+      if in_hash[:name]
+        in_hash[:name] = adjust_country_name(in_hash[:name])
+
+        @country_names_to_update.each_pair do |key, value|
+          if existing = self.where("lower(name) like ?", key.downcase).first
+            existing.update_attributes(:name => value)
+          end
+        end
```

Comment:
I think this `from_hash` function is called on each entry in the csv file. So that means that this loop through the `@country_names_to_update` is run many extra times.